tests/typecheck_examples.tests

```diff
@@ -0,0 +1,1007 @@
+examples/distribution/clientToClientViaServerAP.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/distribution/clientToServerDeleg.links
+examples/distribution/clientToServerDeleg.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/distribution/serverToClient.links
+examples/distribution/serverToClient.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/distribution/serverToClient2.links
+examples/distribution/serverToClient2.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/distribution/simpleServerAP.links
+examples/distribution/simpleServerAP.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+ignore : broken example
+
+Typecheck example file examples/document-replace.links
+examples/document-replace.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/draggable.links
+examples/draggable.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/draggableDb.links
+examples/draggableDb.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/draw.links
+examples/draw.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/extensible-case.links
+examples/extensible-case.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/factorial.links
+examples/factorial.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/ffi.links
+examples/ffi.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/filter.links
+examples/filter.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/formsTest.links
+examples/formsTest.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/games/breakout.links
+examples/games/breakout.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/games/pacman.links
+examples/games/pacman.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/games/tetris.links
+examples/games/tetris.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+ignore : broken example
+
+Typecheck example file examples/games/twentyfortyeight.links
+examples/games/twentyfortyeight.links
+filemode : args
+args : --config=tests/typecheck_examples.tests.config
+
+Typecheck example file examples/handlers/alert.links
```

Comment:
Do not add the `handlers/` directory to the test suite. Those examples that I want to maintain are already included in `handlers.test`.

Reply:
If examples in `handlers/` and `tofix/` are not being maintained, maybe they are candidates for deletion?

Reply:
Yes... I need to clean up `handlers/`.

Reply:
Fun fact: The tofix directory hasn't been touched in 9 years. I suggest we just get rid of it.